addOns/domxss/src/main/java/org/zaproxy/zap/extension/domxss/DomXssScanRule.java

```diff
@@ -453,9 +453,12 @@ private void scanHelper(String attackVector, String url) throws DomAlertExceptio
         }
 
         List<WebElement> inputElements;
+        List<WebElement> buttonElements;
 
         try {
             inputElements = findHelper(driver, By.tagName("input"));
+            buttonElements = findHelper(driver, By.tagName("button"));
+            inputElements.addAll(buttonElements);
```

Comment:
You can combine this into just one line:
`inputElements.addAll(findHelper(driver, By.tagName("button")))`

Reply:
Will do. I'll combine

Reply:
combined